firefly_client/firefly_client.py

```diff
@@ -119,6 +119,7 @@ def __init__(self, host=_my_localhost, channel=None, basedir='firefly', html_fil
         self.url_bw = protocol + '://' + self.this_host + '/%s%s?__wsch=' % (self._basedir, self.html_file)
         self.listeners = {}
         self.channel = channel
+        self.headers = {'FF-channel':channel} if channel else {}
```

Comment:
To make sure channel is always set, can we use the default channel=uuid.uuid1() rather than channel=None?